ddtrace/llmobs/_experiment.py

```diff
@@ -212,5 +249,54 @@ def _run_task(
         self._llmobs_instance.flush()  # Ensure spans get submitted in serverless environments
         return task_results
 
-    def _run_evaluators(self, task_results, raise_errors: bool = False) -> None:
-        pass
+    def _run_evaluators(self, task_results: List[_TaskResult], raise_errors: bool = False) -> List[_EvaluationResult]:
+        if not task_results:
+            raise ValueError("No task results to evaluate.")
```

Comment:
is this actually an error? i feel like it's expected usage? i'd rather it just try to run an empty list rather than doing more

Reply:
Fair point, no need for raising I guess 😄